@app/server/src/middleware/installPostGraphile.ts

```diff
@@ -183,6 +184,14 @@ export function getPostGraphileOptions({
 
       // Adds custom orders to our GraphQL schema
       OrdersPlugin,
+
+      // Plugins for just Development mode
+      ...(!isDev
+        ? []
+        : [
+            // updates the `pg-database-smart-tags.schema.json` file based on the database
+            CreateJsonSchemaPlugin,
+          ]),
```

Comment:
Avoiding the negative makes this easier to read IMO.

```suggestion
      ...(isDev
        ? [
            // updates the `pg-database-smart-tags.schema.json` file based on the database
            CreateJsonSchemaPlugin,
          ]
        : []),
```